completions/xdg-mime

```diff
@@ -25,9 +25,9 @@ _comp_cmd_xdg_mime()
     local cur prev words cword comp_args
     _comp_initialize -- "$@" || return
 
-    local args
-    _count_args
-
+    local ret
+    _comp_count_args
+    local args=$ret
```

Comment:
Would it not be better to just use `ret` below, or was it considered too generic a name from the location it is set, or...?

Reply:
In the original code, the variable `args` has been used also by lines 42, 46, and 53 in addition to line 31. There are no other function calls that affect the variable `ret`, so we can technically replace all of these references to `args` with `ret`. However, it is not so obvious that `ret` is not affected by other codes in between, and it might be broken in the future. For this reason, I basically give a distinct variable name when the variable is referenced by the codes far from the position where it is obtained.